install.sh

```diff
@@ -159,6 +160,9 @@ legacy_transformations_start() {
         cp "$INSTALL_DIR/autoupdate/registered" "$INSTALL_DIR/registered" || LEGACY_TRANSFORM_FAILURES="true"
     fi
 
+    # Check if we need to move .shared files entries to `--local githooks.shared`
+    # and split comma-separated `--global githooks.shared`
+
```

Comment:
I think this comment is a leftover?

Reply:
+1 would be good to remove to avoid confusion when we look at it in the future.

Reply:
Done.